projects/go-lib/src/lib/components/go-table/go-table.component.scss

```diff
@@ -259,12 +258,13 @@
   background: transparent;
   border: 0;
   flex: 1;
+  line-height:1.3;
```

Comment:
Looks like we're missing a space before the value here. 😬

Reply:
fixed!